src/component/header.tsx

```diff
@@ -0,0 +1,40 @@
+
+import { Box, Flex, Link, Image } from '@chakra-ui/react';
+
+
+
+export const Header = () => {
+
+  return (
+    <Box   p='24px' paddingLeft='80px'>
```

Comment:
quando você coloca espaçamento de padding 24px, e coloca o paddingLeft 80px, o espaçamento da esquerda vai ficar 24 + 80 px = 104px no total. 

```suggestion
    <Flex   px={20} py={6}>
```

20 equivale a 80px no chakra e 6 equivale a 24px. px é o padding horizantal, ou seja, no eixo de x e o py no eixo de y, seria na vertical.